constants/consts.ts

```diff
@@ -0,0 +1,2 @@
+export const LIST_PLACEHOLDER = require('../assets/list-placeholder.png');
```

Comment:
Should we rename this folder to `utils` or `common` or something to avoid this `constants/consts`? (my eyes are bleeding)

Reply:
[`/constants`](https://docs.expo.dev/get-started/start-developing/#file-structure) is folder convention for expo, but yeah the filename sucks. The URLs file is also a mess.